src/parser/expression.js

```diff
@@ -316,7 +316,7 @@ pp.parseSubscripts = function (base, startPos, startLoc, noCalls) {
     } else if (this.match(tt.backQuote)) {
       let node = this.startNodeAt(startPos, startLoc);
       node.tag = base;
-      node.quasi = this.parseTemplate();
+      node.quasi = this.parseTemplate(true);
```

Comment:
Could do, but it would be a break from the style in the rest of Babylon (and, technically, slightly slower).